types/fabric/fabric-impl.d.ts

```diff
@@ -2236,7 +2236,20 @@ interface IObjectOptions {
 	 */
 	clipTo?: Function;
 
-	/**
+    /**
+     * A fabricObject that, without stroke define a clipping area with their shape. filled in black
+     * the clipPath object gets used when the object has rendered, and the context is placed in the center
+     * of the object cacheCanvas.
+     * If you want 0,0 of a clipPath to align with an object center, use clipPath.originX/Y to 'center'
+     */
+    clipPath?: Object;
```

Comment:
Any reason to use `Object` here? The general guidance is:

> Using the types Function and Object is almost never a good idea. In 99% of cases it's possible to specify a more specific type. Examples are (x: number) => number for functions and { x: number, y: number } for objects. If there is no certainty at all about the type, any is the right choice, not Object. If the only known fact about the type is that it's some object, use the type object, not Object or { [key: string]: any }.

Reply:
Good call @PranavSenthilnathan - should have been fabric.Object not js Object. I've updated.

Reply:
Scratch that. `fabric.` is redundant in this context. `Object` is the fabric Object. That is how the rest of the descriptor references it so i'm going to follow suit and omit the `fabric.`